src/autoComplete/autoCompleteList.tsx

```diff
@@ -123,30 +127,34 @@ extends Component <AutoCompleteListProps, AutoCompleteListState> {
     return (
       <ul
         key={this.props.name}
-        className={cc([prefix({ 'autoComplete-list': true }), this.props.className])}
+        className={cc(['kirk-autoComplete-list', this.props.className])}
         role="listbox"
       >
         {this.props.items.slice(0, this.props.maxItems).map((item, index) => {
-          const status = this.state.selectedIndex === index ? (
-            this.props.selectedItemStatus
-           ) : ItemChoiceStatus.DEFAULT
+          const status =
+            this.state.selectedIndex === index
+              ? this.props.selectedItemStatus
+              : ItemChoiceStatus.DEFAULT
           return (
-            <AutoCompleteListItem
+            <li
+              role="option"
               key={this.props.itemKey(item)}
-              item={item}
-              className={this.props.itemClassName}
-              highlighted={index === this.state.highlightedIndex}
-              status={status}
-              select={this.onSelect(index)}
-              onDoneAnimationEnd={this.props.onDoneAnimationEnd}
+              aria-selected={index === this.state.highlightedIndex}
+              className="kirk-autoComplete-item"
             >
-              <div>
-                {this.props.renderItem({ item, index })}
-              </div>
-            </AutoCompleteListItem>
+              <AutoCompleteListItem
```

Comment:
It's not possible to set the tag `li` on the autocomplete list instead of wrapping it with a `li` ?